packages/lodestar/test/e2e/sync/sync.test.ts

```diff
@@ -9,7 +9,7 @@ describe("syncing", function () {
 
   it("should sync from other BN", async function () {
     this.timeout(0);
-    const bn = await getDevBeaconNode({SECONDS_PER_SLOT: 2, SLOTS_PER_EPOCH: 8}, {}, 8);
+    const bn = await getDevBeaconNode({SECONDS_PER_SLOT: 2, SLOTS_PER_EPOCH: 8}, {sync: {minPeers: 0}}, 8);
```

Comment:
I would keep it with condition to have 1 peer just to see if gossip will enable once it gets peer